pages/detailed-example-workflow/Step_2_homolog_collection.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: default
+title: Homolog collection
+parent: Detailed Example Workflow
+nav_order: 2
+permalink: /detailed-example-workflow/step1-project-dir
+---
+
+# Homolog Collection
+
+1.  Collect putative homologs from input taxa using fisher.py
+
+    `fisher.py [OPTIONS]`
+
+    Optional arguments: 
+    - `-t`, `--threads <N>` Number of threads, where N is an integer
+        - Default : 1
+
+    - `-n`, `--max_hits <N>` Max number of hits to collect, where N is an integer
+
+        - Default: 5
+
+    - `--keep_tmp` Keep temporary files
+
+    - `--all_BBH`  Keep all significant BLAST hits regardless of phylogenetic affiliation as determined with FastTree through the phylogenetically aware route
+
+    - `--add <input_metadata.tsv>` Path to input metadata file(different from original the one in config.ini) that contains new input proteomes. Must be used with –add_to option below.
+
+     - `--add_to <fisher_out>` Previous fisher.py output directory to add to. Must be used with the --add option above.
+
+     - `-o`, `--output <out_dir>` Path to user-defined output directory
+
+        - Default: ./fisher_out_<M.D.Y>
+     - `-h`, `--help`  Show this message and exit.
+
+    Default fisher.py output:
+
+    * a directory called "fisher_out_<M.D.Y>" that contains:
+
+        - fasta files of orthologs from the database with putative homologs  from newly input taxa appended to the end of the fasta file. NOTE: if no putative homologs were found in any newly input proteome for a particular ortholog in the database, that ortholog will not be present in this directory. All files will have the extension “.fas”.
```

Comment:
Are there two spaces between "homologs" and "from"?

Reply:
No.